config.go

```diff
@@ -172,6 +173,15 @@ func (c *config) discoverSingle(glob string, m *map[string]string) error {
 	for _, match := range matches {
 		file := filepath.Base(match)
 
+		// One Windows, ignore any plugins that don't end in .exe.
+		// We could do a full PATHEXT parse, but this is probably good enough.
+		if runtime.GOOS == "windows" && strings.ToLower(filepath.Ext(file)) != "exe" {
```

Comment:
Thanks @mitchellh. That is a much cleaner solution. I have tested your PR at once on a Windows machine if it also solves the Chocolatey problem. But now packer does not find any plugin.

```
C:\Users\vagrant\packer-windows>set PACKER_LOG=debug

C:\Users\vagrant\packer-windows>packer build -debug -only=vmware-iso windows_2008_r2.json
2015/06/11 22:20:31 [INFO] Packer version: 0.7.5
2015/06/11 22:20:31 Packer Target OS/Arch: windows amd64
2015/06/11 22:20:31 Built with Go Version: go1.4.2
2015/06/11 22:20:31 [DEBUG] Ignoring plugin match C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-chroot.exe, no exe extension
2015/06/11 22:20:31 [DEBUG] Ignoring plugin match C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-chroot.exe.ignore, no exe extension
2015/06/11 22:20:31 [DEBUG] Ignoring plugin match C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-ebs.exe, no exe extension
2015/06/11 22:20:31 [DEBUG] Ignoring plugin match C:\ProgramData\chocolatey\lib\packer\tools\packer-builder-amazon-ebs.exe.ignore, no exe extension
...
```

The `filepath.Ext()` returns the file name extension used by path. The extension is the suffix beginning at the final dot in the final element of path; it is empty if there is no dot. From http://golang.org/pkg/path/filepath/#Ext

I have tested it with again after changing the `"exe"` into `".exe"` and packer then worked like a charm!